modules/ocf_rancid/manifests/init.pp

```diff
@@ -49,4 +40,35 @@
     special => 'hourly',
     require => Package['rancid'],
   }
+
+  # Delete all logs older than 14 days.
+  # This would be better with logrotate, but the log files contain the date in
+  # their name, which logrotate doesn't handle very well. The files aren't large
+  # at all, so they don't need to be compressed, there's just a lot of them
+  # since they are created once for every hour.
+  cron { 'clean-rancid-logs':
+    command => 'find /var/log/rancid/* -mtime +14 > /dev/null',
```

Comment:
i like how i read this, thought "wtf use logrotate", then read your comment above and saw you pre-empted me :P

Reply:
Haha yeah, I tried logrotate first but couldn't find a good way to do that didn't involve adding a find command somewhere anyway. If we needed compression too I would have done that, but we just want a bit of cleanup here.